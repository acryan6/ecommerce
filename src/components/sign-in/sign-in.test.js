import { shallow } from 'enzyme';
import React from 'react';
import SignIn from './sign-in.component.jsx';

it('expect to render sign-in component', () => {
  expect(shallow(<SignIn />)).toMatchSnapshot();
})
