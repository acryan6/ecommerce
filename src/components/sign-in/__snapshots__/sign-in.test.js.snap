// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render sign-in component 1`] = `ShallowWrapper {}`;
